site/content/docs/latest/reference/faq.md

```diff
@@ -1,3 +1,108 @@
 # Frequently Asked Questions
 
-TBC
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Will you still be able to decrypt if you no longer have access to your cluster?](#will-you-still-be-able-to-decrypt-if-you-no-longer-have-access-to-your-cluster)
+- [How can I do a backup of my SealedSecrets?](#how-can-i-do-a-backup-of-my-sealedsecrets)
+- [Can I decrypt my secrets offline with a backup key?](#can-i-decrypt-my-secrets-offline-with-a-backup-key)
+- [What flags are available for kubeseal?](#what-flags-are-available-for-kubeseal)
+- [How do I update parts of JSON/YAML/TOML/.. file encrypted with sealed secrets?](#how-do-i-update-parts-of-jsonyamltoml-file-encrypted-with-sealed-secrets)
+- [Can I bring my own (pre-generated) certificates?](#can-i-bring-my-own-pre-generated-certificates)
+- [How to use kubeseal if the controller is not running within the `kube-system` namespace?](#how-to-use-kubeseal-if-the-controller-is-not-running-within-the-kube-system-namespace)
+- [How to verify the images?](#how-to-verify-the-images)
+- [How to use one controller for a subset of namespaces](#how-to-use-one-controller-for-a-subset-of-namespaces)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Will you still be able to decrypt if you no longer have access to your cluster?
+
+No, the private keys are only stored in the Secret managed by the controller (unless you have some other backup of your k8s objects). There are no backdoors - without that private key used to encrypt a given SealedSecrets, you can't decrypt it. If you can't get to the Secrets with the encryption keys, and you also can't get to the decrypted versions of your Secrets live in the cluster, then you will need to regenerate new passwords for everything, seal them again with a new sealing key, etc.
+
+## How can I do a backup of my SealedSecrets?
+
+If you do want to make a backup of the encryption private keys, it's easy to do from an account with suitable access:
+
+```console
+$ kubectl get secret -n kube-system -l sealedsecrets.bitnami.com/sealed-secrets-key -o yaml >main.key
```

Comment:
Are we choosing to use leading `$` for commands or not? I see uses of both alternatives.

Reply:
I think the non-written rule is to use

Reply:
Then we should change all the other instances in these two files to use, I think.

Reply:
We don't seem to be following that convention in these files, we should fix that.

Reply:
I guess this one should be shell as it has no outputs